packages/skeleton-svelte/src/lib/components/Progress/Progress.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+	import type { ProgressProps } from './types.js';
+
+	let {
+		value,
+		min = 0,
+		max = 100,
+		// Root
+		base = 'overflow-x-hidden bg-surface-200-800',
+		height = 'h-2',
+		rounded = 'rounded',
+		classes = '',
+		// Indicator
+		indicatorBase = 'bg-primary-500 h-full',
+		indicatorRounded = 'rounded',
+		indicatorTransition = 'transition-[width]',
+		indicatorIndeterminateAnimation = 'indeterminate',
+		indicatorClasses = ''
+	}: ProgressProps = $props();
+
+	const indeterminate = $derived(value === undefined);
+	const fillPercentage = $derived(indeterminate ? 50 : ((value! - min) / (max - min)) * 100);
+	const width = $derived(`${fillPercentage}%`);
```

Comment:
for reactive variables, it seems that we should prefix it with rx, see the docs https://next.skeleton.dev/resources/contribute/components#dynamic-style-props

Reply:
I think we both mistunderstood when to use `rx`, see: https://discord.com/channels/1003691521280856084/1010694301883318326/1219296077434585198 where Chris explains it more thoroughly

Reply:
Done.